### DISCUSSION TITLE
Recording a free ride without workout?

### DISCUSSION THREAD

**Discussion by MoritzML** [dvmarinoff/Auuki#223] [Q&A] [answered]
Hi there,

just stumbled over your app and just want to say thank you! I wanted something like this and was also thinking of building something like this, but I am mostly having Frontend experience, so this might have been a long path for me :D

I have tried it today for the first time, but after an hour of testing out different things I was not able to **record my session without starting a workout**. I saw another question about the difference between the two play buttons but apparently I am only having 1 play button which does both, starting the workout and recording. Is there another way now to do it, or is it maybe even a bug?
(using mac and edge)

<img width="1676" alt="Bildschirmfoto 2024-12-31 um 17 09 50" src="https://github.com/user-attachments/assets/1ae945c1-9d61-4c20-a4d6-f0b8b0f5d539" />

Thanks in advance and best regards!

**Comment by dvmarinoff** [comment, marked as answer]
Hey,

I decided to remove the two play buttons in favor of leaving only the `start workout button`. It was confusing for Users to have 2 buttons. So this feature is no longer available.

But if you skip to the end of any workout the recording will continue in free form mode. The trainer will switch to slope mode with grade value of 0%, and the counter should start to count up for the current interval since it is open ended now.

Currently what you can do is: create a custom workout with just one short interval, you can even skip the interval with the lap button and record the rest of the session in free form mode.

There is a build in editor at the workouts tab, or you can use any other `.zwo` editor.

**Reply by puremana** [reply]
Two play buttons might have been confusing, but no free ride option is equally confusing. My suggestion would be to have an option above "Library" on the workout tab that says something like "Free Ride" with an infinite symbol in the column for minutes.

Then you could select that and press play, which should act like the play button you removed

**Comment by Lory3881** [comment]
Another question by me: yesterday I tried to attend a structured workout in no-erg mode just setting the smart trainer not as controllable but only as power meter. All went good except that speed and distance were not recorded (cadence was) even though the relative button was ON. Is it normal ? Is it another correct way to do a "sim" mode structured workout ?

**Reply by dvmarinoff** [reply]
I just ran a test with the following configuration:

Tacx Flux trainer connected as Power Meter, with Power, Cadence and Speed switched on for the Power Meter
Simulation set to Speed

<table>
<tr>
<td>
<img width="320" alt="Screenshot 2025-01-04 at 13 52 16" src="https://github.com/user-attachments/assets/bd4e7619-768d-456c-959b-4cb0a365b22c" />
</td>
<td>
<img width="320" alt="Screenshot 2025-01-04 at 13 52 47" src="https://github.com/user-attachments/assets/07ef0aeb-d19c-4894-883e-18421b5a206b" />
</td>
</tr>
</table>

And here is how the .FIT file looks:

<img width="1440" alt="Screenshot 2025-01-04 at 13 53 07" src="https://github.com/user-attachments/assets/ce69cd05-6f43-474c-8aaa-cc28b3ad80e7" />

The app always records distance and speed if there is power or speed source connected.
But it is possible to achieve a configuration in which power/speed/distance is only 0s.
For example if Power or Speed are not switched on for the currently connected device.
But if that happens power/speed/distance won't change in the interface too.

It's either this or a 3rd party application is stripping the data while trying to recalculate speed and distance (Garmin and Strava may do that).

Let me know if I can help further!

**Reply by Lory3881** [reply]
You're right! It's been probably caused by an app running in background which was "draining" those data. Tried few minutes ago and speed was correctly shown.
Thank you so much for help
